### PR TITLE
re: Remove usage of `lazy_static!` in `runtime`

### PR DESCRIPTION
**Pull request by pmnoxx**
Let's remove usage of `lazy_static!` in runtime.

Part of #531

### CODE REVIEW

**Comment by pmnoxx** [comment]
@mm-near Can you take a look?

File: runtime/runtime/Cargo.toml [hunk 1]
```diff
@@ -13,7 +13,7 @@ serde_json = "1.0"
 log = "0.4"
 tracing = "0.1"
 rand = "0.7"
-lazy_static = "1.4"
+once_cell = "1.5.2"
```
Comment (mm-near) [inline runtime/runtime/Cargo.toml:16]: out of curiosity - why did we update ? (just to have the newest version or was there something wrong with the old one?)

File: runtime/runtime/src/lib.rs [hunk 2]
```diff
@@ -439,7 +439,7 @@ impl Runtime {
                 )?;
             }
             Action::DeleteAccount(delete_account) => {
-                near_metrics::inc_counter(&metrics::ACTION_DELETE_ACCOUNT_TOTAL);
+                (metrics::ACTION_DELETE_ACCOUNT_TOTAL.inc());
```
Comment (mm-near) [inline runtime/runtime/src/lib.rs:442]: do we need the surrounding brackets?

**Review by mm-near — APPROVED** [review]

File: runtime/runtime/Cargo.toml (hunk 1 above)
Comment (pmnoxx) [reply runtime/runtime/Cargo.toml:16]: @mm-near No, we are still using the old library. The newest version is 1.8.0

**Review by chefsale — APPROVED** [review]

**Pull request merged**